back/exercises3.sh

```diff
@@ -0,0 +1,30 @@
+# !/bin/bash
+echo -n "ファイル名を入力=> "
+read file_name
+echo -n $file_name"で良いですか？良ければy、ダメならn => "
+read answer
+if [ $answer = "y" ] ; then  # 答えがyes
+  if [ -e $file_name ]; then
+    echo -n "次の中から計算したい項目（sum, avg, min, max）を実行=> "
+    read function
+    if [ $function = "sum" ] ; then  # 答えがsum
+      awk '{sum+=$1}END{print sum}' $file_name
+    elif [ $function = "avg" ] ; then  # 答えがavg
+      awk '{x++;sum+=$1}END {print sum/x}' $file_name
```

Comment:
```suggestion
     awk '{sum+=$1}END {print sum/NR}' $file_name
```
上記の方がシンプルだったりしますかね？？